pkgs/applications/misc/seatd/default.nix

```diff
@@ -27,7 +27,11 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ systemd ];
 
-  mesonFlags = [ "-Dlogind=enabled" "-Dbuiltin=enabled" ];
+  mesonFlags = [
+    "-Dlogind=enabled"
+    "-Dlibseat-logind=systemd"
+    "-Dbuiltin=enabled"
```

Comment:
```suggestion
    "-Dlibseat-logind=systemd"
    "-Dlibseat-builtin=enabled"
```

Reply:
Good catch! I should be more careful checking Meson output.